docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -181,7 +181,7 @@ $ ansible-doc rhoas.rhoas.create_kafka
 === Using the OpenShift Application Services Ansible collection
 
 [role="_abstract"]
-The procedure in this section shows how to create and run an example playbook. The example playbook includes modules for creating a Kafka instance, topic, and service account, and assigning permissions to the service account. You also configure modules to delete the resources that you previously created.
+The procedures in this section show how to create and run an example playbook. The example playbook includes modules for creating a Kafka instance, topic, and service account, and assigning permissions to the service account. You also configure modules to delete the resources that you previously created.
 
 .Prerequisites
```

Comment:
I think there's a couple of structural problems in the doc at the moment. 

If you look at the TOC (http://file.ams.redhat.com/aecollin/JK-10345/README.html) the sections that follow 'Using the OpenShift Application Services Ansible collection' are at the same level as that section, when they should really be subordinate to it. All of those standalone procedures are part of using the collection. So, I think you need to bump all of those headings down to level 4 headings.

Then, you'd need to introduce a new section heading for the very first procedure, because that's kind of free-floating at the moment. There's a couple of things happening in that procedure, but overall, I guess it's about 'Creating a new playbook'?

Reply:
Those comments apply to the Terraform section also.

Reply:
Thanks @jbyrne-redhat. Have amended accordingly.